webui/react/src/components/Searches/Searches.tsx

```diff
@@ -0,0 +1,950 @@
+    };
+  }, [selectAll, selectedExperimentIds, excludedExperimentIds, total]);
+
+  const colorMap = useGlasbey([...selectedExperimentIds.keys()]);
+
+  const experimentFilters = useMemo(() => {
+    const filters: V1BulkExperimentFilters = {
+      projectId: project.id,
+    };
+    return filters;
+  }, [project.id]);
+
+  const numFilters = useMemo(() => {
+    return (
+      Object.values(experimentFilters).filter((x) => x !== undefined).length -
+      1 +
+      rootFilterChildren.length
+    );
+  }, [experimentFilters, rootFilterChildren.length]);
+
+  const handleSortChange = useCallback(
+    (sorts: Sort[]) => {
+      setSorts(sorts);
+      const newSortString = makeSortString(sorts.filter(validSort.is));
+      if (newSortString !== sortString) {
+        resetPagination();
+      }
+      updateSettings({ sortString: newSortString });
+    },
+    [resetPagination, sortString, updateSettings],
+  );
+
+  useEffect(() => {
+    if (!isLoadingSettings && settings.sortString) {
+      setSorts(parseSortString(settings.sortString));
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [isLoadingSettings]);
+
+  const fetchExperiments = useCallback(async (): Promise<void> => {
+    if (isLoadingSettings || Loadable.isNotLoaded(loadableFormset)) return;
+    try {
+      const tableOffset = Math.max((page - 0.5) * PAGE_SIZE, 0);
+
+      // always filter out single trial experiments
+      const filters = JSON.parse(filtersString);
+      filters.filterGroup.children.push({
+        children: [
+          {
+            columnName: 'numTrials',
+            kind: 'field',
+            location: 'LOCATION_TYPE_EXPERIMENT',
+            operator: '>',
+            type: 'COLUMN_TYPE_NUMBER',
+            value: 1,
+          },
+        ],
+        conjunction: 'and',
+        kind: 'group',
+      });
+
+      const response = await searchExperiments(
+        {
+          ...experimentFilters,
+          filter: JSON.stringify(filters),
+          limit: isPagedView ? settings.pageLimit : 2 * PAGE_SIZE,
+          offset: isPagedView ? page * settings.pageLimit : tableOffset,
+          sort: sortString || undefined,
+        },
+        { signal: canceler.signal },
+      );
+      const total = response.pagination.total ?? 0;
+      const loadedExperiments = response.experiments;
+
+      setExperiments((prev) => {
+        if (isPagedView) {
+          return loadedExperiments.map((experiment) => Loaded(experiment));
+        }
+
+        let newExperiments = prev;
+
+        // Fill out the loadable experiments array with total count.
+        if (prev.length !== total) {
+          newExperiments = new Array(total).fill(NotLoaded);
+        }
+
+        // Update the list with the fetched results.
+        Array.prototype.splice.apply(newExperiments, [
+          tableOffset,
+          loadedExperiments.length,
+          ...loadedExperiments.map((experiment) => Loaded(experiment)),
+        ]);
```

Comment:
this might be inherited but we really shouldn't be mutating state in place like this -- it causes issues with the table and selection data, see https://github.com/determined-ai/determined/pull/9020